products/llbuildSwift/BuildSystemBindings.swift

```diff
@@ -18,6 +18,10 @@ import Foundation
 
 import llbuild
 
+#if !LLBUILD_C_API_VERSION_4
+  import llbuild_c_api_version_unsupported
```

Comment:
We can now use `#error("Unsupported API version")`

Reply:
Doesn’t that require Swift 5 (based on the PR I found earlier)?

Sounds good to me, it wasn’t clear if it was going to be available in needed targets.

Reply:
It looks like it was implemented for 4.2: https://github.com/apple/swift-evolution/blob/master/proposals/0196-diagnostic-directives.md

We usually track Swift master but you can maintain compatibility using:

```swift
#if swift(>=4.2)
  #error("Unsupported API version")
#else
  import llbuild_c_api_version_unsupported
#endif
```